octopoes/octopoes/api/router.py

```diff
@@ -308,3 +308,12 @@ def delete_node(
         if e.response.status_code == HTTPStatus.NOT_FOUND:
             raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Node does not exist")
         raise
+
+
+@router.get("/bits/recalculate")
```

Comment:
A `POST` (and not `PUT` or `GET`) endpoint would make more sense in this context?

Reply:
`GET` shouldn't cause side effects, so this should indeed be a `POST`.